doc/package-notes.xml

```diff
@@ -652,4 +652,34 @@ cp ${myEmacsConfig} $out/share/emacs/site-lisp/default.el
 
 </section>
 
+<section xml:id="sec-weechat">
+<title>Weechat</title>
+<para>
+Weechat can currently be configured to include your choice of plugins.
```

Comment:
Explicitly mentioning that this is an expression to install is more helpful, as some other packages are configured via nixpkgs config settings instead.